spec/requests/internal/users_spec.rb

```diff
@@ -51,6 +51,24 @@
     end
   end
 
+  describe "POST internal/users/:id/verify_email_ownership" do
+    it "allows a user to verify email ownership" do
+      post "/internal/users/#{user.id}/verify_email_ownership", params: { user_id: user.id }
+
+      verification_link = "#{ApplicationConfig['APP_PROTOCOL']}#{ApplicationConfig['APP_DOMAIN']}/verify_email_ownership?confirmation_token=#{user.email_authorizations.order('created_at DESC').first.confirmation_token}&username=#{user.username}"
+
+      expect(ActionMailer::Base.deliveries.count).to eq(1)
+      expect(ActionMailer::Base.deliveries.first.subject).to eq("Verify Your #{ApplicationConfig['COMMUNITY_NAME']} Account Ownership")
+      expect(ActionMailer::Base.deliveries.first.text_part.body).to include(verification_link)
+
+      sign_in(user)
+      get verification_link
+      expect(user.email_authorizations.last.verified_at).to be_within(1.minute).of Time.now.utc
+
+      ActionMailer::Base.deliveries.clear
```

Comment:
Not making this a before/after each because this is the only spec messing with mailers.